site/docs/tutorials/abstract-getting-started.mdx

```diff
@@ -0,0 +1,132 @@
+
+### How to make your first design change?
+Abstract follows a pattern of **Branch** → **Edit** → **Merge** which is familiar from software version control systems such as Git. Each Abstract project has its own **Master Branch** that is the single source of truth, the latest accepted design for that project.
+
+To make changes to the Master Branch, you first need to create a new branch for your changes. To create a new branch, press the plus (+) icon next to the "Branches" page link.
+
+<Image src="../../static/tutorials/abstract-getting-started/abstract-create-branch-button.png" alt="Creating a new branch in Abstract" style="width: 100%; max-width:850px; margin-bottom:24px;" viewable/>
+
+Give your branch a descriptive name. Many teams even include a JIRA (or similar software) ticket number in the branch name. For example, the branch name could be "login-page (HDS-123)".
+
+Your newly created branch is a perfect copy of your Master Branch. You can now freely explore design ideas without worrying about changing the Master Branch or messing up other designers' branches. 
+
+To save an edit to your branch, you need to do a **Commit**. Commits can be made in the Abstract desktop app but the easier way is to do them directly from Sketch. When Abstract detects new changes in your Sketch file, a commit popup will appear at the bottom-middle of the Sketch window. Pressing the "Commit Changes" button allows you to write a commit message. The commit message describes what changes the commit includes and it is usually a couple of sentences long at maximum. After writing the commit message, press "Commit Changes" button to finish the commit.
+
+<Image src="../../static/tutorials/abstract-getting-started/sketch-commit-window.png" alt="Sketch commit popup" style="width: 100%; max-width:500px; margin-bottom:24px;" viewable/>
+
+When you have made all changes to the branch, it is time to **Merge** it to the Master Branch. Merging means taking all of your new design changes to combining them with the designs in the Master Branch - creating a new source of truth for your project. To start the merge, press "Merge branch..." (or "Update From Master") button at the top-right corner of the branch screen. You may skip the merge description and press "Merge and Archive" button to continue.
+
+<Image src="../../static/tutorials/abstract-getting-started/abstract-merge-branch-button.png" alt="Merge branch button in Abstract" style="width: 100%; max-width:850px; margin-bottom:24px;" viewable/>
+
+Next, Abstract checks if there are any **Conflicts** in your merge. Conflicts mean that someone else (another designer) has made changes to the Master branch that conflict with the changes you just made. Abstract provides easy-to-use interface to select whether you want to use your changes or the changes in the Master Branch. Make the selection for each artboard/symbol to finalise the merge.
```

Comment:
Abstract provides an easy-to-use interface to select whether you want to use your changes or the changes in the Master Branch. Make the selection for each artboard/symbol to finalize the merge.